addons/web/static/src/core/dialog/dialog_service.js

```diff
@@ -58,14 +58,12 @@ export const dialogService = {
                     id,
                 },
             };
-            if (env.isSmall) {
-                const scrollOrigin = { top: window.scrollY, left: window.scrollX };
-                dialog.dialogData.scrollToOrigin = () => {
-                    if (!Object.keys(dialogs).length) {
-                        window.scrollTo(scrollOrigin);
-                    }
-                };
-            }
+            const scrollOrigin = { top: window.scrollY, left: window.scrollX };
+            dialog.dialogData.scrollToOrigin = () => {
+                if (!Object.keys(dialogs).length) {
+                    window.scrollTo(scrollOrigin);
+                }
+            };
```

Comment:
What about something like this? (untested)

```js
const scrollOrigin = { top: window.scrollY, left: window.scrollX };
dialog.dialogData.scrollToOrigin = () => {
    if (!hasWindowBeenResized && !Object.keys(dialogs).length) {
        window.scrollTo(scrollOrigin);
    }
};

let hasWindowBeenResized = false;
const onResize = () => {
    hasWindowBeenResized = true;
};
window.addEventListener('resize', onResize, { once: true });
```

In the `close` callback:

```js
function close() {
    window.removeEventListener('resize', onResize);
   // ...
}
```

:bulb: As we cache the scroll offset, it might be better to trigger the scroll only if the dimensions of the page have not changed. If the user resizes the page, the elements might take up more vertical space, and the `scrollToOrigin` call may scroll the view to the wrong location. With this code, we only trigger the scroll if the user has not resized the page. Would that be better?

Note: Maybe I'm thinking too much

Reply:
Hello @julien-banken 
actually i have talked this issue with aab so he suggested to simply remove the condition 
but if you want we can try your way :blush:

Reply:
Ok, we can keep it as is to keep things simple :smiley: My suggestion might make the code unnecessarily complex: aab said that switching from the desktop to the mobile view is not always well supported in Odoo. So, we don't necessarily have to support that here. I would ask the JS team to do the final review as /web is not really in our scope.